src/sprout/Helpers/Register.php

```diff
@@ -604,4 +605,37 @@ public static function getDisplayConditions()
         return self::$display_conditions;
     }
 
+
+    /**
+     * Register search handler
+     *
+     * @param string $class Controller to register which implements the
+     *        FrontEndSearch interface. Must be fully namespaced.
+     * @param string $table The name of the keywords table, e.g. page_keywords
+     * @param array $where Optional list of where clauses @see SearchHandler->addWhere()
+     * @return void
+     */
+    public static function searchHandler($class, $table, $where = [])
+    {
+        $handler = new SearchHandler($table, $class);
+
+        if (!empty($where) and count($where) > 0) {
+            foreach ($where as $clause) {
+                $handler->addWhere($clause);
+            }
+        }
+
+        self::$search_handlers[] = $handler;
+    }
+
+
+    /**
+     * Return list of SearchHandler objects
+     *
+     * @return array List of SearchHandler instances
+     */
+    public static function getSearchHandlers()
+    {
+        return self::$search_handlers;
```

Comment:
Check for 'sprout.search_handlers' config and array_merge with registered handlers.
In the future we'll replace this with an exception to tell people to upgrade.